src/Microsoft.DocAsCode.YamlSerialization/YamlSerializer.cs

```diff
@@ -130,7 +130,7 @@ private IEventEmitter CreateEventEmitter()
             }
             else
             {
-                return new TypeAssigningEventEmitter(writer, IsOptionSet(SerializationOptions.Roundtrip));
+                return new TypeAssigningEventEmitter(writer, IsOptionSet(SerializationOptions.Roundtrip), new Dictionary<Type, string>());
```

Comment:
Docfx uses `SerializationOptions.DisableAliases` option, it should be safe to pass empty dictionary for `tagsMapping`.

[Click here to check related change in YamlDotNet](https://github.com/aaubry/YamlDotNet/commit/0ce8ac1c508c5b36bec665da83932f5be134cdcb#diff-a785a783f91bff1967fd01a1c63a6fb7ef5a7278f99b84945b3961a4eef9089d)